### DISCUSSION TITLE
'ArrowSmallRightIcon' is deprecated.ts(6385)

### DISCUSSION THREAD

**Discussion by goustaff** [tailwindlabs/heroicons#1149] [Help]
Hello,
I would like to know what should be used instead of `<ArrowSmallRightIcon />` now.

It is tagged as deprecated, but VSCode doesn't tell me how to stay updated.

I think it could be nice to JSDoc better those deprecations notes.

For instance, if I write `window.pageYOffset`:
![image](https://github.com/tailwindlabs/heroicons/assets/118654172/af773099-ea42-450c-8b9a-17504aaf6d04)

> @deprecated
> This is a legacy alias of scrollY.

Thank you in advance.

**Comment by tordans** [comment]
In addition, it would be great to show the deprecation in the https://github.com/tailwindlabs/heroicons/blob/master/CHANGELOG.md

**Comment by BuckAMayzing** [comment]
Just noticed this myself, and have noticed that it still has no resolution. As for the arrow icons, it  looks like they may have been incorrectly marked as deprecated when replacing `arrow-sm-*` with `arrow-small-*`, as there don't appear to be any direct analogs. Is this right?

**Comment by Nybkox** [comment]
@gustaveWPM  is was moved to `ArrowRightStartOnRectangleIcon` (now there are `ArrowRightStartOnRectangleIcon` and `ArrowRightEndOnRectangleIcon`)
